src/DataMiner/SLNetTypes/Info/IDObjects/AlarmTreeID.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Skyline.DataMiner.Net.Messages;
+using Skyline.DataMiner.Net.Messages.SLDataGateway;
+using System;
+
+namespace Skyline.DataMiner.Net
+{
+    /// <summary>
+    /// Represents a reference to one specific alarm tree.
+    /// </summary>
+    [Serializable]
+    public sealed class AlarmTreeID //: DMAObjectRef, IEquatable<AlarmTreeID>
+    {
+        // default ctor is required for protobuf
```

Comment:
Is this correct, or is "ctor" a typo?

Reply:
It is correct. 'ctor' is often used as a shorthand for 'constructor'.

I copied these parts over from the official source.
Perhaps we should use the full word for public documentation though. 
Up to you

Reply:
indeed, I'll adapt this to constructor so it's clear for everyone